bcgov/moh-prime
# Feature/prime 2055 effective license lookup

This PR cannot be merged until the refactor is done.

To test:

- launch app and look at the lookup api call.
- using a database tool adjust the effective date of a license's detail to the future
- refresh the page and investigate that call
- see that it is no longer shown.
